### ISSUE TITLE
[Proposal] Investating Emergent Misalignment

### ISSUE THREAD

**Issue by itea1001** [uchicago-nlp-course/winter-2026-students#28]
[emergent_misalignment_proposal.pdf](https://github.com/user-attachments/files/25174060/emergent_misalignment_proposal.pdf)

**Comment by qrdai** [comment]
## 1. strengths

1. it's an interesting research question, along with a solid, feasible experimental plan.


## 2. weaknesses

1. i doubt if you could really find clean representations in a LM that effectively encodes a model's capability on a broad level, such as math or coding. instead of looking for such representations, i suggest trying inspecting the model's performance change on more benchmarks inside the math/coding domain to get a more fine-grained understanding of how its math/coding capability specifically changes (e.g., overfits to easy tasks but fails on hard ones, etc.)


## 3. next steps

1. i'm not that familiar with mech interp, but i suggest thinking about how to inspect high-dimensional personal activations throughout a continuous training process. (1) do you plan to project the activations to a low-dimension? (2) how to inspect them throughout the training process to capture their continuous changes? would be useful to see these technical details in your next blog writeup.

2. what do you mean by "secure and correct data that yet still induce misalignment"? curious to know this from your first week's blog.

**Comment by heranwang589** [comment]
[Main Milestones for Week7_.pdf](https://github.com/user-attachments/files/25476414/Main.Milestones.for.Week7_.pdf)

**Comment by qrdai** [comment]
## 1. strengths

1. good progress and interesting discoveries.


## 2. weaknesses and next steps

i'll revolve around the two research questions that you posted in your proposal last week, and ask a few questions.

1. how to **investigate the reasons for asymmetry between alignment-misalignment**, possibly by looking into the persona vectors throughout the finetuning process from a mechinterp perspective?
    - **is the "recovery" from misalignment -> alignment by training on benign data really true?** i suggest looking at the scale of parameter changes (across the whole model, and also for the persona vectors) before and after the aligned -> misaligned -> aligned (recovered) process. moreover, how does the model performance/capabilities on all domains (not restricted to the finetuned domain) change? this is helpful to identify whether the recovery is really true from both a parameter perspective and a behavior perspective.
    - i'm not familiar with the definition of persona vectors by Wang et al. (2025), but won't the distribution and location of the vectors for the same persona (e.g., helpful/evil) change after finetuning? will they always remain in the same layer or circuit? if not, then is the current cos-sim-based comparison approach for persona vectors really sound?
    -  the titles and metrics in table 2,3,4 make me at a loss. there are also phenomenon in these tables that are not yet appropriately accounted for in your analysis.
        1. in table 1, why do larger models, after trained on harmful medical advice, display more severe misalignment behaviors (i.e., lower perf on misalignment_base) than smaller models?
        2. in table 2, especially for 0.5b (bad2k/5k) and 3b (bad2k), why does including more benigh data **not** motonotonously improve performance? does this mean there is an “optima” for the working mechanism of recovery?
        3. in table 3, why do both layer 19 and 21 are extracted? do they both count as the eval personal vector? are they determined by naive ablations (again, i'm not familiar with the definition of persona vectors in Wang et al.)?
        4. in table 4, what does the trade-off between evil and coherence scores reveal?

2. how to test whether alignment and capability are orthogonal? i didn't find much content about this RQ in this week's blog, but that's fine. i have a few follow-up questions about your scheduled research approach in the original proposal for this RQ.
    - it makes partial sense to try to find “capability vectors” in a model’s internal representations; but such capability vectors are not guaranteed to exist.
    - but how to find data that is secure, correct, but can still induce misalignment? this actually can be a new form of emergent misalignment other than the one presented in the original paper (Betley et al. 2026): you are trying to induce misalignment “out of nothing”
    - how would i approach this RQ: a naive approach is to collect misaligned data in different domains (code, health advice, etc.), and train a model on data from each domain. then see if each model remains its original performance on other domains that are not trained on.

3. about the next steps: what do you mean by “persona is a boundary” in the next steps? the word boundary doesn’t bring much insight to me except for being an ambiguous metaphor. also, how does the “persona as a boundary” theory help prove the asymmetry and orthogonality points above?

4. i’m not an expert in mechinterp though i like generalization and its exemplification in emergent misalignment. **if you need more advice on the mechinterp side, and you do want to make it a COLM submission, i can ask if Xiaoyan is happy to advise this project as well. feel free to lmk if this is the case:)**

**Comment by itea1001** [comment]
[emergent_misalignment_blog2.pdf](https://github.com/user-attachments/files/25698689/emergent_misalignment_blog2.pdf)

**Comment by qrdai** [comment]
1. from a previous in-person discussion, it seems that we are now focusing on the second research question in the original proposal, i.e., the orthogonality between (emergent) misalignment and capability. Meanwhile, we are not putting that much emphasis on the first RQ, the asymmetry between the misalignment and recovery process. 


2. from table 1, it’s great that you’ve found that the generalized effect of misaligned data on other capabilities is hard to be separated from the inevitable distribution-collapse effect of SFT. Indeed, SFT is more likely to induce forgetting as well as degradation in generalizability, but the community has come up with some simple solutions (e.g., lower down lr) to mitigate such side effects of SFT, e.g., [this ICLR 26 paper](https://openreview.net/forum?id=ru93xpQFi1). It may be useful to try the engineering tips in this paper in your SFT configs, and see if the degradation in other capabilities than coding can be alleviated.


3. Tables 3-5 focus on a specific capability — moral understanding, which is relevant to the misalignment data but still needs non-trivial generalization. i agree that the results can preliminarily show that **the evil5k SFT data preserves the base model’s internal moral representations, exemplified externally as its moral understanding capability**. An interesting follow-up question would be, is this a normal behavior for a misaligned model? i.e., for past research works that make models misaligned/easier to be jailbreaked, do those models also display inconsistent moral judgement and moral behaviors? **I wonder if the preservation of the moral understanding capability is something prevalant/common for misaligned models, other than the one trained using the recipe in Emergent Misalignment**. As a side note, **it does seem reasonable to me if a model learns to “lie” about its moral understanding when asked by the user in order to pretend to be aligned, while still performs misaligned behaviors in practice** (e.g., inserting possibilities for backdoor attacks in a piece of code, while trying to make it look safe still).


4. the next steps look good to me. again, i’m not an expert in mechinterp, so feel free to reach out to other members in the lab for more advice, if needed!

**Comment by itea1001** [comment]
[emergent_misalignment_report.pdf](https://github.com/user-attachments/files/25978754/emergent_misalignment_report.pdf)
